src/test/java/org/assertj/core/internal/classes/Classes_assertHasPackage_Test.java

```diff
@@ -0,0 +1,103 @@
+package org.assertj.core.internal.classes;
+
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.error.ShouldHavePackage.shouldHavePackage;
+import static org.assertj.core.error.ShouldNotBeNull.shouldNotBeNull;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+import java.util.stream.Stream;
+
+import org.assertj.core.internal.ClassesBaseTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+@DisplayName("Classes assertHasPackage")
+class Classes_assertHasPackage_Test extends ClassesBaseTest {
```

Comment:
Could you split this into two separate test classes, one for each assertion method? (e.g., `Classes_assertHasPackage_with_Package_Test` and `Classes_assertHasPackage_with_String_Test`)

Reply:
Sure I'll do it. About the naming convention, as I said in the other comment, I didn't find "with" to separate the arguments in other tests around, can you confirm this? Or if this is a new convention we're introducing I'll use it for sure.

Reply:
Sorry, my bad I missed those tests following the convention. I'll rename my classes immediately, ignore the other comments!